pkg/util/util.go

```
@@ -25,3 +29,29 @@ func ParseInt64(s string) (int64, error) {
 	}
 	return strconv.ParseInt(s, 10, 64)
 }
+
+// FileExist 判断文件是否存在
+func FileExist(file string) bool {
+	stat, _ := os.Stat(file)
+	if stat == nil {
+		return false
+	}
+	return true
+}
+
+// CheckIsDir 判断文件类型是否是目录
+func CheckIsDir(file string) (bool, error) {
+	stat, _ := os.Stat(file)
```

do not ignore any error